src/routes/rules/components/RulePreviewDialog.vue

```diff
@@ -73,3 +73,10 @@ const preloadImage = () => {
   img.src = props.imageUrl;
 };
 </script>
+
+<style>
```

Comment:
All style blocks should be scoped to prevent bleeding across components 
```suggestion
<style scoped>
```